be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -184,16 +187,34 @@ Status FragmentExecutor::prepare(ExecEnv* exec_env, const TExecPlanFragmentParam
                 } else {
                     scan_operator->set_io_threads(nullptr);
                 }
+                for (int32_t j = operators.size() - 1; j >= 0; --j) {
```

Comment:
Using `operators` after moved in L182 is undefined behavior. Maybe we could move L190~L198 before L179.